tests/support/helpers.lua

```diff
@@ -16,6 +16,10 @@ function M.setup_test_env()
 
   -- Change to temporary directory
   vim.cmd('cd ' .. temp_dir)
+  
+  -- Clear buffer and load plugin as part of setup
+  M.clear_buffer()
```

Comment:
now that these two calls were added, the rest of the function is in a different level of abstraction. So extract a function to deal with the current working directory.